microsoft-edge/dualengine/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Introduction to Microsoft DualEngine
+description: Host web content in your Win32 apps with the Microsoft DualEngine interface.
+author: MSEdgeTeam
+ms.author: edgededev
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.localizationpriority: high
+ms.technology: windows-integration
+ms.date: 11/06/2023
+---
+# Introduction to Microsoft DualEngine API
+
+The Microsoft DualEngine API allows you to embed and control a Internet Explorer within your app.
+
+The DualEngine API differs from similar APIs like the Web Browser Control by providing a browsing experience

```

Comment:
```suggestion
The DualEngine API differs from similar APIs like the Web Browser Control by providing a browsing experience that's 
```

Reply:
The DualEngine API provides a browsing experience that's equivalent to a normal Internet Explorer tab, to provide increased compatibility with legacy websites.

The DualEngine API differs from similar APIs like the Web Browser Control, and requires distinct development work.